### ISSUE TITLE
zooming on mobile

### ISSUE THREAD

**Issue by michelsondan** [kingdido999/zooming#66]
1. Is there a way to zoom with 2 fingers touch and focusing on a area.

2. When zooming, How to aviod auto close so I can continue to move element, and closing zomming just 
 on another finger click.

Thank you

**Comment by kingdido999** [comment]
1. It does not support multi-touch functionality at this moment.
2. We're not able to disable auto close after we release mouse/finger on the image, but maybe we can add an option to disable auto close.

**Comment by Giovancruz** [comment]
any update on disable auto close?

**Comment by BenFictional** [comment]
Yes, it would be ideal if users on mobile could drag the image around without closing it. 
I've trying to display the image larger than the mobile viewport and allow users to explore it via swiping.
